types/cassanknex/index.d.ts

```diff
@@ -100,7 +100,7 @@ declare namespace CassanKnex {
         uuid <K extends keyof T>(columnName: K): TypeMatchedValue<T, K, types.Uuid, this>;
         varchar <K extends keyof T>(columnName: K): TypeMatchedValue<T, K, string, this>;
         list <K extends keyof T>(columnName: K, typeName: string): TypeMatchedValue<T, K, any[], this>;
-        primary(primaryKey: string): this;
+        primary(primaryKeys: string | string[]): this;
```

Comment:
Seems like this should be varargs, given the JS source

```suggestion
        primary(...primaryKeys: (string | string[])[]): this;
```